src/index.ts

```diff
@@ -58,6 +59,9 @@ program
   .option('--compile-cairo')
   .option('--no-compile-errors')
   .option('--check-trees')
+  // for development mode
+  .option('--dev', 'Run AST sanity checks on every pass instead of the final AST only', false)
+  .option('--no-format-cairo', "Don't format cairo output")
```

Comment:
Use `'` instead of `"` for the option message in order to keep the same format.

Reply:
You're right, the only reason it's different is because the use of a `'` inside the string were the formatter prefer "Don't" against 'Don\'t'.  Nonetheless when changed form `no-format-cairo` to `format-cairo` this will be solved

Reply:
👌